### ISSUE TITLE
Possible figure for paper

### ISSUE THREAD

**Issue by ha0ye** [weecology/forecasting-course#20]
paragraph 2 of "General instructional design" in `paper.md` :
> The instructors then lead a group discussion on the paper, guiding the students through the discussion questions and integrating mini-lectures where appropriate to address common points of confusion about the paper (e.g., walking through a complicated modeling approach).

To help explain the linkage between the discussion of the paper and mini-lectures, I created a possible diagram with some hypothetical topics:
https://docs.google.com/drawings/d/1mspzqPr8J_nV4ILirX1m9Yl2IBwbOxX1jMFf6hOEk30/edit?usp=sharing

@ethanwhite and @skmorgane should be able to edit directly; and everyone should have commenting access.

**Comment by ethanwhite** [comment]
Thanks for this @ha0ye! We spent some time with it today and while we really like it in concept we eventually came to the conclusion that we should probably not include it. Our reasoning for this is two fold:

1. While we do actively use this mini-lecturing approach in the class it isn't actually something that's actively included as part of the OER. I.e., there are no links anywhere to mini-lectures. As such, making this the subject of the only figure in the paper feels like it really emphasizes something that is more of an aside from the perspective of the current OER (and one that might result in us being asked to develop a bunch of written material that we aren't currently ready to create).
2. Since this is about the actual OER, not a conceptual version of how we would build one, we think that instead of hypothetical examples if we're going to include this it should be actual examples from the course. We started changing the topics to actual examples, but while we loved the idea of spreading things out by stage of the modeling approach like you've done that actually made it more difficult to do this because they tend to be clustered in material from a small number of those stages.

So, if it's OK with you, we'd like to proceed without the figure for now.
